test/Microsoft.AspNetCore.Mvc.TagHelpers.Test/InputTagHelperTest.cs

```diff
@@ -1053,6 +1053,7 @@ public void Process_GeneratesFormattedOutput_ForDateTime(string specifiedType, s
                     { "TEXT", null, "text" },
```

Comment:
Need checks like `Editor_FindsCorrectDateOrTimeTemplate_ForDateTimeNotRfc3339(...)` and `Editor_FindsCorrectDateOrTimeTemplate_ForDateTimeNotRfc3339(...)` somewhere in here. Fine to limit that to the `week` input type.

Reply:
Separately, add tests to confirm `asp-format` and `ModelMetadata.EditFormatString` (usually set with `[DisplayFormat(ApplyFormatInEditMode = true, DataFormatString = "format string")]`) override the default formats for both `week` and `month`.

Reply:
> override the default formats for both week and month

There already are tests for that in `DefaultEditorTemplatesTest` here https://github.com/aspnet/Mvc/pull/7045/files/bb95452869050d39c95c96fa5e328fef6031fd25#diff-8b48a4e50a9006318eff0699fe1cafc9R1094

I'll add them to `InputTagHelperTest` though